extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -147,6 +157,32 @@ export class BaseSlides extends BaseCarousel {
         this.autoplayDelay_);
   }
 
+  /**
+   * Called on user interaction to toggle the autoplay feature.
```

Comment:
Called by `toggleAutoplay` action to ...

Reply:
Done